### PR TITLE
Views

### PR DESCRIPTION
**Pull request by LuisSalas94**
**Hello Microverse Team**
_In this branch I:_
- Made sure that there were no linter errors
- Made sure that you used the correct Gitflow
- Made sure that you documented your work in a professional way
- Used preprocessed HTML file with embedded Ruby code.
- Used layouts and templates for shared content
- Implemented the design from the sneak peek wireframes
- Used methods I created
  

**Happy Codding**

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -1,5 +1,10 @@
 class PostsController < ApplicationController
-  def index; end
+  def index
+    @user = User.find(params[:user_id])
```
Comment (whiz25) [inline app/controllers/posts_controller.rb:3]: - please create a variable to fetch the 5 most recent comments for a given post.

**Review by whiz25 — CHANGES_REQUESTED** [review]
Hi @LuisSalas94, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Nice work 👏🏿 👏🏿 👏🏿 
- Most requirements met
- Code looks clean

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

### Martin Afani
#### Slack: whiz25

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by LuisSalas94** [comment]
**Hello Microverse Team**

I could implement the required change from views/posts/index.html.erb. Now it shows the 5 most recent comments for a given post.

**Happing Coding**

**Review by mahbubul14 — APPROVED** [review]
# Project Approved :trophy: :tada: 

Hi @LuisSalas94 , 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linter error.
:heavy_check_mark: Descriptive readme file.
:heavy_check_mark: Good PR.

### Optional suggestions :arrow_down_small: 

- [ ] The commit message should be written in the imperative tense. `Ex: "Fix bug" instead of "Fixed bug".` [For more details.](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**